pkgs/by-name/ma/mangojuice/package.nix

```
@@ -0,0 +1,69 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  cmake,
+  vala,
+  pkg-config,
+  makeBinaryWrapper,
+
+  gtk4,
+  libadwaita,
+  glib,
+  libgee,
+
+  mangohud,
+  mesa-demos,
+  vulkan-tools,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mangojuice";
+  version = "0.7.8";
+
+  src = fetchFromGitHub {
+    owner = "radiolamp";
+    repo = "mangojuice";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-EWpXikyO7N2NjONqnTx8+9w16Pt5ne7qX67bYirShjc=";
+  };
+
```

```suggestion

  strictDeps = true;

```
Good for cross comp since we need some native and host versions of dependencies at once (like glib)